api/src/test/java/marquez/db/SearchDaoTest.java

```diff
@@ -205,12 +214,16 @@ public void testSearch_resultsWithTimeSorting() {
   }
 
   /** Returns search results grouped by {@link SearchResult.ResultType}. */
-  private List<List<SearchResult>> groupResultsByType(@NonNull List<SearchResult> results) {
+  private Map<SearchResult.ResultType, List<SearchResult>> groupResultsByType(
+      @NonNull List<SearchResult> results) {
     final Map<Boolean, List<SearchResult>> resultsByType =
         results.stream()
             .collect(
                 Collectors.partitioningBy(
                     result -> result.getType() == SearchResult.ResultType.JOB));
```

Comment:
Why not make this a `Map<SearchResult.ResultType, List<SearchResult>>` and just partition by `SearchResult::getType`?

Reply:
Sure, let me simplify this logic 👍